site/specs/numbers.json

```diff
@@ -12424,58 +12423,105 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Add a site to the account</p>\n",
-        "operationId": "POST /accounts/{accountId}/sites",
+        "description": "Add a site to the account",
+        "summary": "Create Site",
+        "operationId": "CreateSite",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
+            "description": "User's account ID",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "1234567"
           }
         ],
         "requestBody": {
           "content": {
             "application/xml": {
+              "schema": {
+                "title": "CreateSite",
+                "properties": {
+                  "Name": {
+                    "description": "The name of the site. Max length restricted to 10 characters",
+                    "type": "string"
+                  },
+                  "Description" : {
+                    "description": "Customer provided description of the site",
+                    "type": "string"
+                  },
+                  "Address" : {
+                    "description": "Service Address for the site",
+                    "type": "object"
+                  },
+                  "CustomerProvidedId" : {
+                    "description": "Customer can provide an optional id (max 10 digits). Note that the customer can use the same id across multiple orders.",
```

Comment:
We could use the `maximum` field instead of putting `max 10 digits` in the description